conf/core.sql

```diff
@@ -10,6 +10,6 @@ CREATE TABLE if not exists core_agent (id integer primary key AUTOINCREMENT, paw
 CREATE TABLE if not exists core_chain (id integer primary key AUTOINCREMENT, op_id integer, host_id integer, ability integer, jitter integer, command text, cleanup text, score integer, status integer, decide date, collect date, finish date, UNIQUE(op_id, host_id, command));
 CREATE TABLE if not exists core_parser (id integer primary key AUTOINCREMENT, ability integer, name text, property text, script text, UNIQUE(ability, property) ON CONFLICT IGNORE);
 CREATE TABLE if not exists core_attack (attack_id text primary key, name text, tactic text, UNIQUE(attack_id));
-CREATE TABLE if not exists core_fact (id integer primary key AUTOINCREMENT, property text, value text, score integer, blacklist integer, source_id text, link_id integer, UNIQUE(source_id, property, value) ON CONFLICT IGNORE);
+CREATE TABLE if not exists core_fact (id integer primary key AUTOINCREMENT, property text, value text, score integer, group_id integer, blacklist integer, source_id text, link_id integer, UNIQUE(source_id, property, value) ON CONFLICT IGNORE);
```

Comment:
change "group_id" to "set_id"-- since we already have a group_id in the groups table, i think it may get confusing.

Reply:
Ok. I changed it and pushed.